AUTHORS

```diff
@@ -0,0 +1,15 @@
+# This is the official list of HTSJDK-Next authors for copyright purposes.
+#
+# The AUTHORS file lists the copyright holders.
+# For example, Broad Institute employees are not listed here, 
+# because Broad Institute holds the copyright.
+
+# This file should be updated whenever a new contributor merges their first commit.
+
+# Names should be added to this file as
+# Organization <webpage>
+# Individual Name <email address or website>
+
+# Please keep the list sorted alphabetically, companies/organizations first, individuals second.
+Broad Institute, Inc. <www.broadinstitute.org>
+Fulcrum Genomics <www.fulcrumgenomics.com>
```

Comment:
Add `Daniel Gómez Sánchez <daniel.gomez.sanchez@hotmail.es>`. If the symbols in my surname are problematic, they can also me removed.

Reply:
I think we should be able to have your name with the proper accents in it in the AUTHORS file.  It was problematic in javadoc I think, but should be fine here.